dev/RadioButtons/RadioButtons.cpp

```diff
@@ -73,19 +73,17 @@ void RadioButtons::OnGettingFocus(const winrt::IInspectable&, const winrt::Getti
             {
                 if (auto const oldFocusedElement = args.OldFocusedElement())
                 {
-                    if (auto const oldFocusedElementAsUIElement = oldFocusedElement.try_as<winrt::UIElement>())
+                    auto oldElementParent = winrt::VisualTreeHelper::GetParent(oldFocusedElement);
```

Comment:
@jevansaks  - Two different RadioButtons cannot have the same RadioButton right? Why was the old if-case failing then?

Reply:
There is a bug in ItemsRepeater where repeater will look at the virtualization info attached property on the element and report its index without also checking that it is the owner of that element and the source of that attach property. So in this case repeater1 is asked what is the index of RadioButton1 which repeater 2 attached a property to saying the index was 0. Thus repeater 1 is reporting 0 instead of checking the owner field on that property and correctly reporting -1.

@jevansaks do we have an issue opened for this already?

Reply:
No I didn't file an issue yet.

Reply:
I've opened #1671